### ISSUE TITLE
layman: Overlay "games-overlay" could not be found in the remote lists

### ISSUE THREAD

**Issue by powerman** [hasufell/games-overlay#77]
```
# layman -s games-overlay
...
 * Succeeded:
 * ------
 * Successfully synchronized overlay "games-overlay".
 * 
 * 
 * Warnings:
 * ------
 * Overlay "games-overlay" could not be found in the remote lists.
 * Please check if it has been renamed and re-add if necessary.
 * 
```

**Comment by nado** [comment]
eselect repository list|grep game 
  [141] gamerlay

So `eselect repository enable gamerlay` should be enough.

**Comment by powerman** [comment]
@nado gamerlay and games-overlay are different overlays. I've no idea is they related in some way or not, but I've currently enabled both and only games-overlay contains ebuild for Baldur's Gate I 1.3.x (I need 1.3.x and not 2.3.x available in main portage because only 1.3 has good enough unofficial Russian translation).

**Comment by hasufell** [comment]
Maybe it has been removed from the layman list. Read https://github.com/hasufell/games-overlay#adding-the-overlay

**Comment by powerman** [comment]
Yes, it was removed, and this is what I've reported - sorry if this wasn't obvious. Probably you should contact layman's list maintainers and ask why your overlay was removed.

**Comment by hasufell** [comment]
> Probably you should contact layman's list maintainers and ask why your overlay was removed.

I don't really care. I let other people maintain this overlay, however. So "issues" barely help. Instead, just provide pull requests, do stuff, contact list maintainers.

**Comment by powerman** [comment]
I've own overlay registered at layman several years ago, and AFAIR they need overlay author's contact email and confirmation. So, I'm not sure someone else may register your overlay in layman's list.

**Comment by hasufell** [comment]
see https://gitweb.gentoo.org/data/api.git/commit/files/overlays/repositories.xml?id=d8a0194ec53d9342a17758ba076ad7fb74afa25f 
and
https://bugs.gentoo.org/626918

if you want to fix those issues, do so, create a bug, then contact @mgorny or whatever to re-add

**Comment by powerman** [comment]
Thanks @hasufell.
